Include <strings.h> for strcasecmp.  The _XOPEN_SOURCE macro
definition overrides _DEFAULT_SOURCE and disables the declaration
in <string.h>.

This avoids an implicit function declaration and build failures with
future compilers.

Submitted upstream: <https://github.com/mate-desktop/engrampa/pull/477>

diff --git a/src/file-utils.c b/src/file-utils.c
index a967cb4aad47971d..03024f32d6739bbb 100644
--- a/src/file-utils.c
+++ b/src/file-utils.c
@@ -27,6 +27,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <strings.h>
 #include <ctype.h>
 #include <time.h>
 #include <unistd.h>
